packages/twenty-server/src/engine/twenty-orm/custom.workspace-entity.ts

```diff
@@ -136,4 +140,20 @@ export class CustomWorkspaceEntity extends BaseWorkspaceEntity {
   @WorkspaceIsNullable()
   @WorkspaceIsSystem()
   timelineActivities: TimelineActivityWorkspaceEntity[];
+
+  @WorkspaceField({
```

Comment:
For custom objects, searchVector column is created by service based on labelMetadataIdenfierFieldMetadataId.

Here in the workspace entity file we don't have access to that potential labelMetadataIdenfierFieldMetadataId (if Im not mistaken). 
Running sync-metadata on existing workspaces will create the searchVector column based on the field "name" which is the default label metadata identifier field. 
Another option would be not to have anything in this file and create a command to backfill existing workspaces, which we could do later since for now we don't have usecases. But this adds manual action while very little custom objects have a different (not name) label identifier I suppose

Reply:
those are very wise words.